### PR TITLE
More information on solctest.

### PR DESCRIPTION
**Pull request by rocky**
* Note useful options
* Link to boost reference.

### Checklist
- [x] Code compiles correctly
- [x] All tests are passing
- [x] New tests have been created which fail without the change (if possible)
- [x] README / documentation was extended, if necessary
- [ ] Changelog entry (if change is visible to the user)
- [x] Used meaningful commit messages

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -71,15 +71,15 @@ Running the compiler tests
 ==========================
 
 The ``./scripts/tests.sh`` script executes most Solidity tests and
-runs ``aleth`` automatically if it is in the path, but does not download it,
+runs ``aleth`` automatically if it is in the path. The script does not download it,
```
Comment (Marenz) [inline docs/contributing.rst:74]: I am not sure that's true, I mean there is literally a `download_aleth` function in there

Comment (rocky) [reply docs/contributing.rst:74]: Ok - I am not going on knowledge here. I was just tyring to clarify what I _thought_ the intent is.

So please suggest something. Or better, _change_  this to what is more accurate. Thanks.

Comment (Marenz) [reply docs/contributing.rst:74]: Yeah, it reads as if it was wrong in the first place

Comment (Marenz) [reply docs/contributing.rst:74]: Thank you for finding this

Comment (chriseth) [reply docs/contributing.rst:74]: `download_aleth` does not always download aleth.

Comment (Marenz) [reply docs/contributing.rst:74]: Okay, I just checked and tried. The behavior is that it does NOT download aleth except when it's run in the CI. So it is basically correct :) We could add that it does not download it except in the CI, but probably not important for contributors

Comment (rocky) [reply docs/contributing.rst:74]: So change 
> The script does not download 

to 

> The script does not always download 

?

Comment (Marenz) [reply docs/contributing.rst:74]: I would say the way you have it now is fine. The CI detail is not important for contributors

**Review by Marenz — DISMISSED** [review]

**Review by Marenz — APPROVED** [review]

**Review by chriseth — APPROVED** [review]

**Pull request merged**